### PR TITLE
test: add null column test case

### PR DESCRIPTION
**Pull request by Yuan325**
Add integration tests to check for `null` columns. ref #757

### CODE REVIEW

File: tests/tool.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func RunToolGetTest(t *testing.T) {
 }
 
 // RunToolInvoke runs the tool invoke endpoint
-func RunToolInvokeTest(t *testing.T, select1Want, invokeParamWant string) {
+func RunToolInvokeTest(t *testing.T, select1Want, invokeParamWant, invokeParamWant2 string) {
```
Comment (duwenxin99) [inline tests/tool.go]: Can we rename `invokeParamWant2` to be something more descriptive such as `invokeParamWantNull`?
